common-npm-packages/packaging-common-v3/nuget/NuGetToolGetter.ts

```diff
@@ -110,13 +110,20 @@ function pathExistsAsFile(path: string) {
 // Based on code in Tasks\Common\MSBuildHelpers\msbuildhelpers.ts
 export async function getMSBuildVersionString(): Promise<string> {
     const msbuild2019Path = 'C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/MSBuild/Current/Bin/msbuild.exe';
+    const msbuild2022Path = 'C:/Program Files/Microsoft Visual Studio/2022/Enterprise/MSBuild/Current/Bin/msbuild.exe';
```

Comment:
It looks like NuGetCommandV2 still uses Common/packaging-common location instead of the npm package (see the package.json: "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",), but because the code is duplicated here I'm updating it here as well.